packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -154,8 +155,8 @@ export class VaDate {
             <va-select
               label="Month"
               name={`${name}Month`}
-              // Remove starting 0 on days less than 10
-              value={month ? month.replace(/^0+/, '') : ''}
+              // Value must be a string
+              value={month ? month.toString() : ''}
```

Comment:
Would `month?.toString()` work instead of having the ternary?

Reply:
Change has been completed